grantnav/csv_layout.py

```diff
@@ -30,13 +32,27 @@
     ("Recipient Org:Location:2:Geographic Code Type", "result.recipientOrganization.0.location.2.geoCodeType"),
     ("Recipient Org:Location:2:Geographic Code", "result.recipientOrganization.0.location.2.geoCode"),
     ("Recipient Org:Location:2:Name", "result.recipientOrganization.0.location.2.name"),
+
+    ("Recipient Individual Id", "result.recipientIndividual.id"),
+    ("Recipient Individual Details:Primary Grant Reason", "result.toIndividualsDetails.primaryGrantReason"),
+    ("Recipient Individual Details:Secondary Grant Reason", "result.toIndividualsDetails.secondaryGrantReason"),
+    ("Recipient Individual Details:Grant Purpose", "result.toIndividualsDetails.grantPurpose"),
+
     ("Funding Org:Identifier", "result.fundingOrganization.0.id"),
     ("Funding Org:Name", "result.fundingOrganization.0.name"),
     ("Funding Org:Postal Code", "result.fundingOrganization.0.postalCode"),
+    ("Funding Org:Charity Number", "result.fundingOrganization.0.charityNumber"),
+    ("Funding Org:Company Number", "result.fundingOrganization.0.companyNumber"),
+
+
     ("Grant Programme:Code", "result.grantProgramme.0.code"),
     ("Grant Programme:Title", "result.grantProgramme.0.title"),
     ("Grant Programme:URL", "result.grantProgramme.0.url"),
-    ("Funding Type:Title", "result.fundingType.0.title"),
+
+    ("Regrant Type", "result.regrantType"),
+
+    ("Funding Type Title", "result.fundingType.0.title"),
```

Comment:
Did you mean to remove the colon  `Funding Type:Title` -> `Funding Type Title` ? I guess it's not really a category since there is only the one field

Reply:
I see further down that you do split by `:`, hence removing

Reply:
Yeah being more strict on the ":" being consistent way to represent the object hierarchy. This also helps with flatten/unflatten processing